pcsx2/GameDatabase.cpp

```diff
@@ -325,6 +326,118 @@ bool GameDatabaseSchema::isUserHackHWFix(GSHWFixId id)
 	}
 }
 
+u32 GameDatabaseSchema::GameEntry::applyGameFixes(Pcsx2Config& config) const
+{
+	// Only apply core game fixes if the user has enabled them.
+#ifdef PCSX2_CORE
+	const bool apply_auto_fixes = config.EnableGameFixes;
+#else
+	const bool apply_auto_fixes = config.EnablePatches;
```

Comment:
i think you meant to do this ?
```suggestion
	const bool apply_auto_fixes = config.EnableGameFixes;
```

Reply:
The Wx version doesn't have `EnableGameFixes`, `EnablePatches` controls both patches and Gamefixes

The Qt version (which uses PCSX2_CORE) introduced `EnableGameFixes`, hence the ifdef statement

Reply:
Could either just do it at the call site instead (VMManager), or make the whole routine `#ifdef PCSX2_CORE`, since it's not used in wx.